Deu erro no flake8.

flake8...................................................................Failed
- hook id: flake8
- duration: 11.87s
- exit code: 1
l10n_br_account/models/account_invoice.py:422:25: E741 ambiguous variable name 'l'

Acho que tem que trocar o nome da variável para line

--- a/l10n_br_account/models/account_invoice.py
+++ b/l10n_br_account/models/account_invoice.py
@@ -397,6 +397,21 @@ class AccountInvoice(models.Model):
         for invoice in self:
             if invoice.document_type_id:
                 if invoice.issuer == DOCUMENT_ISSUER_COMPANY:
+                    if (
+                        not invoice.comment_ids
+                        and invoice.fiscal_operation_id.comment_ids
+                    ):
+                        invoice.comment_ids |= self.fiscal_operation_id.comment_ids
+
+                    for line in invoice.invoice_line_ids:
+                        if (
+                            not line.comment_ids
+                            and line.fiscal_operation_line_id.comment_ids
+                        ):
+                            line.comment_ids |= (
+                                line.fiscal_operation_line_id.comment_ids
+                            )
+
                     invoice.fiscal_document_id._document_date()
                     invoice.fiscal_document_id._document_number()
 

